scripts/gen

```diff
@@ -74,17 +74,17 @@ if [[ "$1" == "post-deploy" ]] && ([[ "$2" == "-a" ]] || [[ "$2" == "--all" ]] |
         shift
     fi
 
-    gen.py deploy --ssh-keyscan $@
+    gen.py post-deploy --ssh-keyscan $@
     if [ $? -ne 0 ]; then
         exit
     fi
 
-    gen.py deploy --gather-mac-addr $@
+    gen.py post-deploy --gather-mac-addr $@
     if [ $? -ne 0 ]; then
         exit
     fi
 
```

Comment:
Also need to include:
gen.py post-deploy --lookup-interface-names $@

I forgot to add this to 'gen' in my commit.

Reply:
np.  I'll ad it